lib/statesman/adapters/active_record.rb

```diff
@@ -76,7 +76,8 @@ def last(force_reload: false)
       end
 
       def reset
-        remove_instance_variable(:@last_transition)
+        remove_instance_variable(:@last_transition) \
+          if instance_variable_defined?(:@last_transition)
```

Comment:
Nit, would generally prefer the full `if` block here rather than escaping the line break - I think this is one of the cops in gc_rubocop too

```suggestion
        if instance_variable_defined?(:@last_transition)
          remove_instance_variable(:@last_transition)
        end
```